packages/coinjoin/src/utils/http.ts

```diff
@@ -10,6 +10,31 @@ export interface RequestOptions extends ScheduleActionParams {
     userAgent?: string;
 }
 
+const skipPascalCasePatch = ['Type', 'BackenMajordVersion', 'LegalDocumentsVersion'];
```

Comment:
typo or there is really field named `BackenMajordVersion`?
```suggestion
const skipPascalCasePatch = ['Type', 'BackendMajorVersion', 'LegalDocumentsVersion'];
```

Reply:
ok https://github.com/zkSNACKs/WalletWasabi/blob/18edb38d4fb0a3ce75ff161e47d4b50fa9f71575/WalletWasabi/Backend/Models/Responses/VersionsResponse.cs#L9 🙈